damage_pyro_volcanyro.sp

```diff
@@ -189,9 +189,9 @@ stock GiveGiantPyro(client)
 		CreateRoboWeapon(client, "tf_weapon_jar_gas", 1180, 6, 1, 2, 0);
 		CreateRoboWeapon(client, "tf_weapon_fireaxe", 348, 6, 1, 2, 0);
 
-		CreateRoboHat(client, FirewallHelmet, 10, 6, 0.0, 1.0, -1.0); 
-		CreateRoboHat(client, GoogolGlassEyes, 10, 6, 0.0, 1.2, -1.0); 
-		CreateRoboHat(client, RailSpikes, 10, 6, 0.0, 0.75, -1.0); 
+		CreateRoboHat(client, FeatheredFiend, 10, 6, 1315860, 1.0, -1.0); 
+		CreateRoboHat(client, DeitysDress, 10, 6, 1315860, 1.2, -1.0); 
+		CreateRoboHat(client, PyromancersMask, 10, 6, 1315860, 0.75, -1.0); 
```

Comment:
with scale `0.75` the mask clips into the normal robot face
`1.0` seems to fit better